content/en/docs/setup/additional-setup/dual-stack/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Install Istio on a Dual-Stack Kubernetes Cluster
+description: Install and use Istio in Dual-Stack mode running on a Dual-Stack Kubernetes cluster.
+weight: 70
+aliases:
+    - /docs/setup/kubernetes/install/dual-stack
+    - /docs/setup/kubernetes/additional-setup/dual-stack
+keywords: [dual-stack]
+owner: istio/wg-networking-maintainers
+test: yes
```

Comment:
We have tests set to `yes` which generated the snips file, but we don't have any tests.